### DISCUSSION TITLE
Améliorer la gestion des emails contact, animation et WTTJ

### DISCUSSION THREAD

**Discussion by fannyblanc** [betagouv/cap#9] [Besoin]
### Besoin

```bash
Nous recevons plusieurs emails par jour sur nos boites mail partagées. Le problème est que nous ne savons pas qui est en charge de les traiter et que du coup, il arrive que nous répondions en même temps.
```

### Solutions envisageables

```bash
1. Est-ce qu'un outil de ticketing pourrait nous aider ?
```

### Informations complémentaires

- [X] J'avais déjà entendu ce besoin au sein de la communauté beta.gouv
- [X] Je me porte volontaire pour participer à la mise en place de la future solution choisie

**Comment by revolunet** [comment]
Chatwoot en cours de test avec @sneko

**Reply by sneko** [reply]
@fannyblanc @AmelAlili , j'attends vos retours sur l'outil :) https://chatwoot.incubateur.net/

**Reply by AmelAlili** [reply]
@sneko les 1ers retours sur chatwoot sont positifs. 
mais il reste le soucis du mail de l'expéditeur qui n'est pas affiché puisqu'il y a une redirection.

**Comment by revolunet** [comment]
toujours d'actu @fannyblanc ?

**Comment by revolunet** [comment]
utilisation de crisp au final
